CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+date: YYYY-MM-DD 23:59
+comments: true
+author: Author(s) name(s)
+published: true
+authorIsRacker: true
+#
+# The *authorAvatar* and *bio* entires are optional, but include them if you can!
+# The avatar must be a hosted image, such as a gravatar.
+#
+authorAvatar: 'https://www.gravatar.com/avatar/<insert hash for your headshot>'
+bio: "<insert a sentence or two about yourself in first or third person>"
+categories:
+    - This Category
+    - That Category
+    - Other Category
+#
+# Use canonical entry if you are republishing a blog from another site, such as
+# your personal blog.  Do  NOT republish without the autor's explicit permission.
+#
+canonical: https://original-url.link.com/post-name/
+metaTitle:
+metaDescription:
+ogTitle:
+ogDescription:
+#
+# The following properties are OPTIONAL and affect the text and image that
+# appear by default in link previews when sharing blog posts.
+#
+ogImage:
+twitterCreator: "@your_twitter_handle" # NOTE: The quotes are required!
+twitterDescription:
+twitterTitle:
+---
+```
+
+**NOTE:** The "ogImage" _must_ be a fully-qualified URL. If you'd like to use an
+image asset that is being uploaded as part of your blog post, the pattern for the
+final URL is as follows:
+
+  `<rackspace CDN>/<image name>-<sha256 hash>.<image extension>`
+
+Where:
+
+  * `<rackspace CDN>` is `https://657cea1304d5d92ee105-33ee89321dddef28209b83f19f06774f.ssl.cf1.rackcdn.com`
+  * `<image name>` is the case-sensitive name of the image _not including the extension_.
+  * `<sha256 hash>` is the 64-character hex output from running the command `sha256sum /path/to/image`
+  * `<image extension>` is `jpg`, `png`, etc.
+
+Example:
+
+  * `https://657cea1304d5d92ee105-33ee89321dddef28209b83f19f06774f.ssl.cf1.rackcdn.com/default-og-image-46fb3587dedfdf950188fabbddd596d67e6b699374a7f4e36b43046d7a24fd09.jpg`
+
+If you'd like to use an image asset from your post that _shouldn't_ appear in
+the post itself, you can include `style="display: none"` on the `<img />` tag
+to hide it within the post while triggering the necessary plugin code to ensure
+it is made available on the CDN based on the preceding naming convention.
+
+Make sure that the dates in the file name and front-matter match.
+
+Available categories include the following:
+
+- Ansible
+- Architecture
+- Automation
+- AWS
+- Azure
+- Chef
+- Cloud Files
+- Cloud Monitoring
+- Cloud-networks
+- Cloud Servers
+- Configuration Management
+- Database
+- Developers
+- DevOps
+- Docker
+- Events
+- General
+- Java
+- Jclouds
+- Jenkins
+- Mailgun
+- Neutron
+- NodeJS
+- OpenStack
+- Oracle
+- Orchestration
+- OSAD
+- Private Cloud
+- Python
+- Salesforce
+- SDK
+- Security
+- SQL Server
+
+If no category fits, use *General*.
+
+If you'd like to use a category that is not in the list, please send an email
+to infodev@rackspace.com. To avoid being flooded with categories, which might apply
+to only one or two blogs, we have automated throttling. However, notify us so
```

Comment:
maybe rephrase this because I don't think we really have "automated throttling"

Reply:
The current system definitely has auto-throttling.  You cannot add a category unless a magic threshold of blogs already has that category.  For example, if I want to use "Microsoft" as a category, I have to find ten or so posts and add the category all at once.  If I do them individually, I get an error.

Reply:
If we elect not to enforce category use, lines 152-155 go away.  We can still leave the suggested category list as a recommendation.

Reply:
Hugo has no implementation that I'm aware of for "auto-throttling" of categories. Suggest we review these categories as part of our redesign and information architecture work.